pom.xml

```diff
@@ -149,6 +152,7 @@
         <version>0.8.7</version>
         <configuration>
           <excludes>
+            <exclude>/src/main/java/it/gov/pagopa/apiconfig/selfcareintegration/model/**/*</exclude>
```

Comment:
🤔 is it needed?
See the exclusions on GHA `code_review.yaml`

Reply:
just an oversight from local test, will be removed.